index.js

```diff
@@ -32,7 +33,7 @@ module.exports = function (options) {
   })
 
   server.on('listening', function () {
-    logger.info(this.address(), 'listening')
+  	logger.info(this.address(), 'listening')
```

Comment:
This is fixed now.

I don't know how that happened. My editor detects the indentation using two spaces automatically and even in the `.editorconfig`, this is set. Still, tabs slipped through somehow.